python/ray/serve/_private/application_state.py

```diff
@@ -391,9 +388,7 @@ def deploy_config(
             )
 
     def _get_live_deployments(self) -> List[str]:
-        deps = self._deployment_state_manager.get_deployments_in_application(self._name)
```

Comment:
i am just aware we have this code. So nice we get rid of it.

Reply:
This was introduced in the previous PR, but nice that we can get rid of it promptly after :)